client/components/profilePage/allUsersModal.jsx

```diff
@@ -28,14 +32,19 @@ export default function AllUsers({
           <Col>
             {allUsers.map((user) => {
               return (
-              <div key={user._id}>
-                <span className='thumbBody'><img className='thumbImage' src = {user.photoUrl}/></span>
+              <Row className = 'm-3' key={user._id}>
+                <div className='thumbBody mx-3'>
+                  <img className='thumbImage' src={user.photoUrl}/>
+                </div>
                 <span>{user.username}</span>
-              </div>
-              )
+                <Button className='mx-3' variant='secondary' onClick={() => {addFriend(user._id)}}>
```

Comment:
This addfriend only updates the database and not the client. So I think the new friend would not show up if we went to the all friends page after a button click here. May be something we want to fix by having a global context variable reset along with the axios request.

do later!